removed rat cams

bagel-deficient opened the pull request:
Given that both rat ambassadors have passed, it is unlikely these cams will be used again.  I figured someone should do this.

I edited button-img, camera-img, and cameras.json.  If there are cam specific references elsewhere, I don't know about them.

Nilla and Chips will be missed.  

I have run live cam commands using this version, so I am confident I didn't break anything major.

bagel-deficient committed:
removed rat cams

Given that both rat ambassadors have passed, it is unlikely these cams will be used again.  I figured someone should do this.

I edited button-img, camera-img, and camqeras.json.  If there are cam specific references elsewhere, I don't know about them.

wazix11 commented on public/cameras.json:
Why remove the crow and hank multi cams?

bagel-deficient replied:
Oops. Didn't mean to do that. Fixed.

bagel-deficient committed:
fixed error in cameras.json

previous commit removed cams that should not have been removed from cameras.json
